### PR TITLE
Updated Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
🙂

### CODE REVIEW

File: package/translate/nl.po [hunk 1]
```diff
@@ -1,754 +1,512 @@
...
 # Heimen Stoffels <vistausss@fastmail.com>, 2021.
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: resources-monitor\n"
-"Report-Msgid-Bugs-To: https://github.com/orblazer/plasma-applet-resources-monitor\n"
-"POT-Creation-Date: 2024-04-05 00:29+0200\n"
-"PO-Revision-Date: 2023-05-31 18:44+0200\n"
-"Last-Translator: orblazer\n"
+"Project-Id-Version: resourcesMonitor-fork\n"
```
Comment (orblazer) [inline package/translate/nl.po:8]: The header should keep the new name `resources-monitor` and the updated copyright year.

**Review by orblazer — CHANGES_REQUESTED** [review]
Also the lines like `#: ../contents/config/config.qml:12` should not be changed due to is up to date.

Your translation seem to based on an older version.

**Comment by Vistaus** [comment]
Hello,

That's odd. I used the exact same file as in the last PR, so there should not be anything wrong. I have no idea what happened. I'll look at it again tomorrow. 🙂

**Comment by orblazer** [comment]
This is due to changes i have made between your first commit and now, like i have mentioned here https://github.com/orblazer/plasma-applet-resources-monitor/pull/76#issuecomment-2028910882.

**Comment by Vistaus** [comment]
That would be nice, thanks. 🙂

File: package/translate/nl.po (hunk 1 above)
Comment (Vistaus) [reply package/translate/nl.po:8]: Hmm, this seems like a Poedit bug. :\

**Comment by orblazer** [comment]
Hello,
Since the file look based on 2.x (or at the start of 3.0) i think the best is to restart from the new version and force push + rename of the commit (for prevent multi commit and have conventional commit).

**Comment by Vistaus** [comment]
Alright, everything should be good now. 🙂

File: package/translate/nl.po [hunk 2]
```diff
@@ -160,15 +171,15 @@ msgstr "Hoogte:"
 
 #: ../contents/ui/config/ConfigAppearance.qml:149
 msgid "Spacing:"
-msgstr ""
+msgstr "Afstand:"
 
 #: ../contents/ui/config/ConfigAppearance.qml:157
 msgid "Fill opacity:"
-msgstr "Ondoorzichtigheid vullen:"
+msgstr "Opvulling van ondoorzichtigheid:"
```
Comment (orblazer) [inline package/translate/nl.po:178]: It's possible to have more shorten text to prevent layout issue ?

**Review by orblazer — CHANGES_REQUESTED** [review]

**Comment by orblazer** [comment]
Hello,
Thanks for the update, once the last change is fixed i will make co-commit to have clean commit (without modify header and without commented messages).

File: package/translate/nl.po (hunk 2 above)
Comment (Vistaus) [reply package/translate/nl.po:178]: I could shorten it to “Ondoorzichtigheid opvullen”. It's not as good, but still good enough translation quality-wise.

**Comment by orblazer** [comment]
i have commit it here : https://github.com/orblazer/plasma-applet-resources-monitor/pull/79/commits/33e9a97ce6fd5c639b326019cdb4a93f678cabef could you review the commit ?

**Comment by Vistaus** [comment]
Looks good to me! 👍 🙂

**Pull request closed without merging**